models/logger.go

```diff
@@ -71,3 +71,32 @@ func (l *Logger) Delete() (err error) {
 	}
 	return nil
 }
+
+// LogMessage holds the message payload
+type LogMessage struct {
+	Message   string `json:"message"`
+	Timestamp string `json:"timestamp"`
```

Comment:
Wouldn't setting this closer to the source of origin be better as this could be open to skew if for some reason the message is delayed.